indico/web/client/js/react/forms/index.js

```diff
@@ -17,7 +19,7 @@ export {
   FinalTextArea,
   FormFieldAdapter,
   unsortedArraysEqual,
-} from './fields';
+} = fields; // Fix circular dependency issue
```

Comment:
I don't think this is still needed w/ the change on how we import the Checkbox component in fields.jsx?

Reply:
I'll double-check in the morning.